Parser/tokenizer.c

```diff
@@ -1089,7 +1083,14 @@ syntaxerror(struct tok_state *tok, const char *format, ...)
     if (!errtext) {
         goto error;
     }
-    int offset = (int)PyUnicode_GET_LENGTH(errtext);
+
+    if (col_offset == 0) {
```

Comment:
What if error occurred at the beginning of the line?

Reply:
The starting point for column offsets is 1

Reply:
> What if error occurred at the beginning of the line?

Oh, I was thinking here about line numbers. This should have been `-1`. Thanks for the catch